addons/debugger/icons/performance.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24" style="-ms-transform: rotate(360deg); -webkit-transform: rotate(360deg); transform: rotate(360deg);"><path d="M6 13H2a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1v-8a1 1 0 0 0-1-1zm-1 8H3v-6h2zM22 9h-4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1V10a1 1 0 0 0-1-1zm-1 12h-2V11h2zM14 1h-4a1 1 0 0 0-1 1v20a1 1 0 0 0 1 1h4a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1zm-1 20h-2V3h2z" fill="#4c97ff"/></svg>
```

Comment:
This looks out of place, this icon fits more. If anyone has a better one, please suggest
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 19v-6a2 2 0 0 0-2-2H5a2 2 0 0 0-2 2v6a2 2 0 0 0 2 2h2a2 2 0 0 0 2-2zm0 0V9a2 2 0 0 1 2-2h2a2 2 0 0 1 2 2v10m-6 0a2 2 0 0 0 2 2h2a2 2 0 0 0 2-2m0 0V5a2 2 0 0 1 2-2h2a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2h-2a2 2 0 0 1-2-2z"/></svg>
```

Reply:
Your suggestion icon looks way too bold.

![image](https://user-images.githubusercontent.com/27910867/147843297-04967e96-7dab-4199-8b21-1ac138d8ac5f.png)

Reply:
> Your suggestion icon looks way too bold.
> 
> ![image](https://user-images.githubusercontent.com/27910867/147843297-04967e96-7dab-4199-8b21-1ac138d8ac5f.png)

you can change the color! (also if it still looks too thick, you can change the `stroke-width`

Reply:
Alright here's is what I ended up with, happy with how this looks.
![image](https://user-images.githubusercontent.com/27910867/148009881-845406dd-d407-4953-ba5d-70670d33d362.png)